di/src/main/java/com/duckduckgo/app/di/AppCoroutineScope.kt

```diff
@@ -16,10 +16,10 @@
 
 package com.duckduckgo.app.di
 
-import javax.inject.Scope
+import javax.inject.Qualifier
 
 /** Identifies a coroutine scope type that is scope to the app lifecycle */
-@Scope
+@Qualifier
 @MustBeDocumented
-@Retention(AnnotationRetention.RUNTIME)
+@Retention(AnnotationRetention.BINARY)
```

Comment:
@aitorvs Not 100% understand why we are using Binary instead of runtime. Documentation says 

“```
A qualifier annotation:
is annotated with @Qualifier, @Retention(RUNTIME), and typically @Documented
```."